examples/subscribing-example-app/public/Google/GoogleMapsMarker.js

```diff
@@ -6,15 +6,35 @@ export class GoogleMapsMarker {
         this.current = markerCoordinate;
         this.lastCompassDirection = "N";
         this.marker = new google.maps.Marker({ icon: "/driverN.png", map: map });
+        this.accuracyCircle = new google.maps.Circle({
+          strokeColor: "#FF0000",
+          strokeOpacity: 0.8,
+          strokeWeight: 2,
+          fillColor: "#FF0000",
+          fillOpacity: 0.35,
+          map,
+          center: markerCoordinate,
+          radius: 100,
+        });
         this.map.setZoom(16);
     }
 
+    createAccuracyCircle() {
```

Comment:
Given this is clearly doing the opposite to `hideAccuracyCircle()` then perhaps `show` would be a more appropriate prefix, rather than `create`. 🤔

Reply:
Good point, see 822cfb5